clients/privacy-center/cypress/e2e/consent.cy.ts

```diff
@@ -325,9 +331,32 @@ describe("Consent settings", () => {
                   analytics: true,
                   gpc_test: true,
                 },
+                extraDetails: undefined,
+                fides_string: undefined,
               },
             });
 
+          // TODO- right now we have to use a wait before our cy.waitUntilFidesInitialized() call which is flakey in CI
```

Comment:
we saw a cy.wait working locally, but in CI this sometimes works and sometimes doesn't

Reply:
I don't really like lingering TODOs like this... I'd prefer you delete this commented out block entirely TBH, the GTM events are pretty well exercised elsewhere.